### PR TITLE
Hide accept / decline text on mobile

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description
Enhancement: Hide accept and decline share text on mobile view

Before this PR within the "Shared with you" files list, a text for accept 
and decline was shown next to the associated icon.
These texts were using excessive space in the mobile view and might 
leaving the "State" column not completely visible.
This has been fixed through hiding those not necessary texts with this PR.

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

### Before
image

### After
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by JanAckermann** [comment]
@pmaier1 agree?

File: apps/files/css/mobile.css [hunk 1]
```diff
@@ -74,4 +74,13 @@ table #headerSize, #headerDate, .filesize, .modified, .date {
 	padding-right: 14px;
 }
 
+/* hide text of the share accept action mobile */
+#fileList .action-accept span:last-child {
+		display: none !important;
```
Comment (ChrisEdS) [inline apps/files/css/mobile.css]: The spacing look too wide

File: apps/files/css/mobile.css [hunk 2]
```diff
@@ -74,4 +74,13 @@ table #headerSize, #headerDate, .filesize, .modified, .date {
...
+}
+
+/* hide text of the share reject action mobile */
+#fileList .action-reject span:last-child {
+		display: none !important;
```
Comment (ChrisEdS) [inline apps/files/css/mobile.css]: The spacing look too wide

**Review by ChrisEdS — APPROVED** [review]

**Review by pmaier1 — APPROVED** [review]

**Comment by phil-davis** [comment]
@JanAckermann can you fill in some more information in the top post? Is there an issue to link to that describes why/what was required? Should there be a changelog entry for this?

**Comment by JanAckermann** [comment]
@phil-davis oops, will do ;)

**Pull request merged**